helpers/settings.py

```diff
@@ -112,9 +130,15 @@ def _set_values(self):
                 if wt == QCheckBox:
                     key1, key2 = widget.objectName().split(':')
                     widget.setChecked(config.data[key1][key2])
-                if wt == QSpinBox:
+                    if key1 == 'sharing' and key2 == 'discord_channel' \
+                            and config.data[key1][key2]:
```

Comment:
I wish there was a cleaner way to do this. Honestly it's just to "look pretty" and have one checkbox disable another, but that isn't strictly necessary (and some might consider it a UX anti-pattern).